mock_vehicle_model_user/CMakeLists.txt

```diff
@@ -105,8 +105,8 @@ find_package(catkin REQUIRED COMPONENTS
 ## CATKIN_DEPENDS: catkin_packages dependent projects also need
 ## DEPENDS: system dependencies of this project that dependent projects also need
 catkin_package(
-#  INCLUDE_DIRS include
-#  LIBRARIES ${PROJECT_NAME}
+  INCLUDE_DIRS include
+  LIBRARIES ${PROJECT_NAME}
```

Comment:
I don't see a library named ${PROJECT_NAME} in this file. This line can be removed.